rollup.config.js

```diff
@@ -19,7 +19,13 @@ function config(output) {
             "SQLiteDatabaseClient",
             "Workbook",
             "ZipArchive",
-            "ZipArchiveEntry"
+            "ZipArchiveEntry",
+            "Runtime",
+            "RuntimeError",
+            "Variable",
+            "Module",
+            "Library",
+            "Inspector",
```

Comment:
This is unrelated but I figured nice to change while I’m here so that you don’t see minified names like `zt` in the inspector when working with the runtime